src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -459,10 +459,22 @@ $(function () {
         .on("change mouseup keyup", update_cart_form);
 
     $(".table-calendar td.has-events").click(function () {
-        var $tr = $(this).closest(".table-calendar").find(".selected-day");
-        $tr.find("td").html($(this).find(".events").clone());
-        $tr.find("td").prepend($("<h3>").text($(this).attr("data-date")));
-        $tr.removeClass("hidden");
+        var $grid = $(this).closest("[role='grid']");
+        $grid.find("[aria-selected]").attr("aria-selected", false);
+        $(this).attr("aria-selected", true);
+        $("#selected-day")
+            .html($(this).find(".events").clone())
+            .prepend($("<h3>").text($(this).attr("data-date")));
+    }).each(function() {
+        // check all events classes and set the "winning" class for the availability of the day-label on mobile
+        var $dayLabel = $('.day-label', this);
+        var classes = ['available', 'waitinglist', 'soon', 'reserved', 'soldout', 'continued', 'over'];
```

Comment:
I will look into this

Reply:
I made it so it works like the widget currently – only showing low if all available events are low. It would be a tiny change to make it so, that we add the low-indicator if any available is low.

Reply:
> The logic in the widget is the same as it shows the „best“ status available.

Okay, I was wondering if they have the same interpretation of what "best" is, but I'll take that as a yes ;)

> It adds the low-indicator only if all events are low. IMHO it feels more suitable to add that indicator if at least one event in the list is low – the „green“ informs that there are events available, so it is worth it to click. Adding the indicator if at least one is low also helps to increase the urgency to book now? Should I change that bevahiour (of course in the widget as well) or should I take the logic from the widget to only show low-availability if all are low?

No, I think it is correct the way it is in the widget. Let's say our day has 3 events and consider three scenarios:
- available + available + available → showing available, no argument there
- available + available + low → ?
- available + available + soldout → showing available, no argument there

If the middle scenario would show "low", it's a little weird that it will show "available" again in the third scenario. So I think showing "available" is the logical option.

Reply:
Okay, makes sense. I already made it that way in the code.